staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -64,14 +65,15 @@ type watchCountGetterFunc func(*apirequest.RequestInfo) int
 // NewWorkEstimator estimates the work that will be done by a given request,
 // if no WorkEstimatorFunc matches the given request then the default
 // work estimate of 1 seat is allocated to the request.
-func NewWorkEstimator(objectCountFn objectCountGetterFunc, watchCountFn watchCountGetterFunc, config *WorkEstimatorConfig) WorkEstimatorFunc {
-	estimator := &workEstimator{
-		minimumSeats:          config.MinimumSeats,
+func NewWorkEstimator(
+	objectCountFn objectCountGetterFunc, watchCountFn watchCountGetterFunc, config *apiserver.WorkEstimatorConfiguration,
+) WorkEstimator {
+	return &workEstimator{
+		minimumSeats:          minimumSeats,
```

Comment:
Oh, sorry, I did not notice the introduction of this field in #111181 .  As noted elsewhere, the minimum is not something we want to vary.  We should just keep using the symbolic constant or a literal `1`.

Reply:
Good point. Removed.